imagebuildah/build.go

```diff
@@ -758,6 +767,7 @@ func (b *Executor) Commit(ctx context.Context, ib *imagebuilder.Builder) (err er
 		PreferredManifestType: b.outputFormat,
 		IIDFile:               b.iidfile,
 		Squash:                b.squash,
+		OnBuild:               b.onbuild,
```

Comment:
I don't think this in necessary.  I commented it out and stuff worked fine.

Reply:
Removed